webhook_handler/main.py

```diff
@@ -28,18 +29,23 @@ class EntityEvent(BaseModel):
     entity: Entity
 
 
-# CONSTANTS.
-PENDING_REVIEW = '44dded64-4164-11df-9218-0019bb4983d8'
-APPROVED = '44de097a-4164-11df-9218-0019bb4983d8'
-LIST_DAILIES = '77b9ab82-07c2-11e4-ba66-04011030cf01'
-LIST_DELIVERY = '712f0fee-ead3-11e2-846c-f23c91dfaa16'
+class Statuses(enum.Enum):
```

Comment:
maybe out of scope but perhaps we should rely on names rather then hardcoded ids as they may differ between clients.

Reply:
From a performance point of view I think its better people hardcode their ids, otherwise they need to make a query which is blocking.